sharpy/linear/src/lib_dbiot.py

```diff
@@ -502,11 +507,13 @@ def eval_panel_fast_coll(zetaP, ZetaPanel, gamma_pan=1.0):
     # zetaP=np.array([-0.243,  0.776,  0.037])
 
     ### verify model consistency
-    DPcpp, DVcpp = eval_panel_cpp(zetaP, ZetaPanel, gamma_pan=gamma)
+    DPcpp, DVcpp = eval_panel_cpp(zetaP, ZetaPanel, 1e-6, gamma_pan=gamma) # vortex_radius
```

Comment:
It is just one of those test under "if __name__ == '__main__':". I left 1e-6 because it does not affect anything else and the "passing" criteria for the test are adjusted to 1e-6